### PR TITLE
update the version map between Chaosd and Chaos Mesh

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

minor fix on document by the way

### CODE REVIEW

**Review by cwen0 — APPROVED** [review]
LGTM

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-physical-machine-chaos.md [hunk 1]
```diff
@@ -6,7 +6,12 @@ title: 使用 Chaos Mesh 创建物理机故障
 
 ## PhysicalMachineChaos 介绍
 
-PhysicalMachineChaos 可用于在物理或虚拟机中模拟网络、磁盘、压力、JVM、时间等故障。
+PhysicalMachineChaos 可用于在物理或虚拟机中模拟网络、磁盘、压力、JVM、时间等故障。在使用 Chaos Mesh 的 PhysicalMachineChaos 功能之前，需要在物理机或虚拟机上部署 Chaosd，Chaos Mesh 与 Chaosd 的版本对应关系如下：
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-physical-machine-chaos.md:9]:
```suggestion
PhysicalMachineChaos 用于在物理机或虚拟机中模拟网络、磁盘、压力、JVM、时间等故障。在使用 Chaos Mesh 的 PhysicalMachineChaos 功能之前，你需要在物理机或虚拟机上部署 Chaosd。Chaos Mesh 与 Chaosd 的版本对应关系如下：
```

File: docs/simulate-physical-machine-chaos.md [hunk 2]
```diff
@@ -6,8 +6,12 @@ This document describes how to create PhysicalMachineChaos (physical machine cha
 
 ## Introduction to PhysicalMachineChaos
 
-You can use PhysicalMachineChaos to simulate the faults of network, disk, pressure, JVM, time, and others in physical or virtual machines.
+You can use PhysicalMachineChaos to simulate the faults of network, disk, pressure, JVM, time, and others in physical or virtual machines. Befor using Chaos Mesh's PhysicalMachineChaos, you need to deploy Chaosd in physical or virtual machines, and the version mapping between Chaos Mesh and Chaosd is as follows:
```
Comment (en-jin19) [inline docs/simulate-physical-machine-chaos.md:9]:
```suggestion
You can use PhysicalMachineChaos to simulate the faults of network, disk, pressure, JVM, time, and others in physical or virtual machines. Before using PhysicalMachineChaos provided by Chaos Mesh, you need to deploy Chaosd in your physical or virtual machines. The version mapping between Chaos Mesh and Chaosd is as follows:
```

**Comment by en-jin19** [comment]
The reason for adding “requires-followup" lable: English document about network chaos in "simulate-network-chaos-in-physical-nodes" is to be updated, and the revised part in this PR is going to be updated.

**Comment by WangXiangUSTC** [comment]
@en-jin19 PTAL again

**Review by en-jin19 — APPROVED** [review]
LGTM

**Pull request merged**